### PR TITLE
ICP-7889 innr device name standarization

### PR DESCRIPTION
**Pull request by MAblewicz**
Names Capitalization change for devices:
RB 265
RB 245
RS 225
RB 285 C

### CODE REVIEW

**Comment by MAblewicz** [comment]
@greens @tpmanley @dkirker Please review my PR. It's just deviceJoinName change for some innr bulbs.

**Review by tpmanley — DISMISSED** [review]

**Comment by dkirker** [comment]
I left a comment on the ticket re innr vs. Innr.

**Review by dkirker — DISMISSED** [review]
According to BD the proper text format is "Innr" (with a capital i), so let's go with that.

**Comment by greens** [comment]
@dkirker Really? Weird that it's been this way for so long then...

**Comment by dkirker** [comment]
Hmmm Yeah, I see I had added ones as innr in March. We should definitely respect a partner's trademark. BD did comment on the ticket that "Innr" is correct, so maybe we should take this time to update all of them? I'm not sure if I asked that question to BD at the time, or just "guessed" at it?

**Comment by MAblewicz** [comment]
Thanks Donald :)
I added capitalization changes required by BD.
@dkirker @greens @tpmanley Please review new device names.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -56,8 +56,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWGU10Bulb50AU", deviceJoinName: "Aurora Smart Dimmable GU10"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FEDC", outClusters: "000A, 0019", manufacturer: "Smarthome", model: "S111-201A", deviceJoinName: "Leedarson Dimmable White Bulb A19"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "innr", model: "RB 265", deviceJoinName: "Innr Smart Bulb White"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "innr", model: "RB 245", deviceJoinName: "innr smart candle warm white"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy:59]: Did you mean to remove the RB 245 fingerprint?

Comment (MAblewicz) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy:59]: Thx @tpmanley . I checked inteliJ history and i looks like i removed it by accident.

**Review by greens — APPROVED** [review]

**Pull request merged**